src/main/kotlin/org/rust/ide/typing/assist/RsSmartEnterProcessor.kt

```diff
@@ -36,6 +39,10 @@ class RsSmartEnterProcessor : SmartEnterProcessorWithFixers() {
             val elementType = each.node.elementType
             when {
                 elementType == LBRACE || elementType == RBRACE -> continue@loop
+                each is RsMatchArm && each.parent is RsMatchBody -> {
+                    val psiFactory = RsPsiFactory(psiFile.project)
+                    each.add(psiFactory.createComma())
```

Comment:
If match arm already has comma, the current implementation adds a new one

![2020-03-30 13 34 44](https://user-images.githubusercontent.com/2539310/77904105-a0facb80-728c-11ea-809b-c00954bd2116.gif)